packages/pouchdb-utils/src/guardedConsole.js

```diff
@@ -0,0 +1,8 @@
+function guardedConsole(method) {
+  if (console !== 'undefined' && method in console) {
+    var args = Array.prototype.slice.call(arguments, 1);
+    console[method].apply(null, args);
```

Comment:
This is failing in the browser, fairly sure it needs to be 
 
    console[method].apply(console, args);